k4FWCore/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+################################################################################
+# Package: k4FWCore
+################################################################################
+gaudi_subdir(k4FWCore v1r0)
+
+find_package(podio)
+
+gaudi_install_scripts()
+
+
+gaudi_add_module(k4FWCore
```

Comment:
Hi @vvolkl , I have a trouble with this. In the previous version, there is `gaudi_add_library` and `gaudi_add_module` respectively. The first marco declares the public header, then the k4FWCore header directory will be installed. But in this branch, no header will be installed.
Could you have a look here? Thank you!
Tao

Reply:
Oh, I forgot about that, but it looks like I can also install the headers for the module. Thanks!